---
layout: page
title: Research
permalink: /research
---

<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<style>
.responsive {
  width: 100%;
  height: auto;
}
</style>
</head>
<body>

<img src="https://raw.githubusercontent.com/jeremysutherland/jeremysutherland.github.io/main/assets/img/fields.jpg" class="responsive">

</body>
</html>

I am interested in how the genetic relationships between hosts and their associated microbiomes change across different environments.
----------------------------------------------------------

I investigate topics related to:

* Host-microbe-environment interactions
* Host genetic influence on microbiome composition
* Conservation in microbial niche space 
* Microbial functional redundancy across environments
* Host evolutionary responses to microbes (and vice versa)

________________________________


Skills & Experience
------------------
* Ph.D. in Bioinformatics and Genomics from Penn State University
* Proficient programming skills in Python, R, and UNIX/bash environments
* Demonstrated experience in modeling and analysis of genomic and microbial sequence data
* Excellent knowledge of biological/statistical data analysis methods
* Familiarity with Linux/Unix command line and high-performance computing environments
* Excellent communication, teamwork, and organizational skills
* Background in population/comparative genomics and microbial ecology

________________________________

Software
------
* Package Environments: Rstudio, conda, pip, jupyter-notebooks
* Genome analysis/variant calling: vcftools, bcftools, IGV
* Genome-wide association studies (GWAS): sommer, rrBLUP, statgenGWAS
* Microbiome compositon analysis: phyloseq, microbiome
* Microbiome sequence analysis: dada2, mothur, Qiime2
* Modeling: base-r, sommer, sJSDM, Scikit-Learn, keras, scipy, pytorch, tensorflow
* Phylogenetics: ape, phytools, phylogram, decipher, 
* Raw data processing: base-r, pandas, numpy, cutadapt, FastQC, dplyr, tidyr
* Sequence alignment: BLAST, clustalx, muscle
* Statistics, base-r, vegan
* Visualizations: Matplotlib, ggplot, seaborn

__________________________________

Recent Publications
------------------

[ Sutherland, J., Bell, T., Trexler, R. V., Carlson, J. E., & Lasky, J. R. (2022). Host genomic influence on bacterial composition in the switchgrass rhizosphere. Molecular Ecology, 31, 3934â€“ 3950. https://doi.org/10.1111/mec.16549](https://github.com/jeremysutherland/jeremysutherland.github.io/files/9098613/Molecular.Ecology.-.2022.-.Sutherland.-.Host.genomic.influence.on.bacterial.composition.in.the.switchgrass.rhizosphere.pdf)

All Publications
------------------

[Google Scholar](https://scholar.google.com/citations?hl=en&user=fNlmqGwAAAAJ)
